docs/examples/datasets/pipelines.py

```diff
@@ -350,3 +350,91 @@ def test_multiple_pipelines(client):
     )
     assert df.shape == (4, 4)
     assert found.sum() == 4
+
+
+# docsnip aggregate_with_str_window
+@meta(owner="data-eng-oncall@fennel.ai")
+@dataset
+class ActivityGroupedDataset:
+    timestamp: datetime
+    user_id: int = field(key=True)
+    amount_forever: float
+    amount_1d: float
+
+    @pipeline(version=1)
+    @inputs(Activity)
+    def create_grouped_dataset(cls, activity: Dataset):
+        return (
+            activity.dropnull(columns=["amount"])
+            .groupby("user_id")
+            .aggregate(
+                [
+                    Sum(
+                        of="amount",
+                        window="forever",
+                        into_field="amount_forever",
+                    ),
+                    Sum(of="amount", window="1d", into_field="amount_1d"),
+                ]
+            )
```

Comment:
After a recent change, IIRC, aggregate can take *args without having to actually pass a list. That is, you can probably get rid of `[` and `]`. If you do that, hopefully indentation changes by few spaces and you can stuff in `Sum(...)` in the same line instead of splitting it over multiple lines. 

If it still doesn't fit in a single line, could you please use a slightly shorter field name e.g. `amount_forever` to `sum_forever` or simply `amount`. 

As you can see, we also obsess over text fitting in fewer lines in docs so that it feels simpler to read/write.

Reply:
got it